### ISSUE TITLE
Nox android 12 Zygisk:no Ramdisk:yes

### ISSUE THREAD

**Issue by koto343** [HuskyDG/magisk-files#131]
Hi. I've installed nox-delta on new nox 12 via Magisk.Terminal.Emulator cause via nox delta no availability to install magisk. 

Now I've new situation 

Zygisk:no Ramdisk:yes
![image](https://github.com/HuskyDG/magisk-files/assets/139908236/c76b3e97-19f3-4864-ad0e-242badf46a44)

Zygisk is enabled in settings but it still apears no on magisk main screen

Can anyone help me to solve this problerm? Thx!

**Comment by koto343** [comment]
@HuskyDG Hi! Thx for last update. Last magisk delta c9f9ce03-delta(26105) instaling on nox with no problem but still can't activate zygisk

**Comment by koto343** [comment]
@HuskyDG sorry. Also I've created folder /sbin and nothing changed 
[magisk_log_2023-08-11T12.47.20.log](https://github.com/HuskyDG/magisk-files/files/12320706/magisk_log_2023-08-11T12.47.20.log)

**Comment by VisionR1** [comment]
Yeah, I have the same problem all works good except the zygisk and riru

**Comment by neervanbiqs** [comment]
Same problem here

**Comment by neervanbiqs** [comment]
[magisk_log_2023-08-15T22.53.17.log](https://github.com/HuskyDG/magisk-files/files/12345357/magisk_log_2023-08-15T22.53.17.log)
![Screenshot_1](https://github.com/HuskyDG/magisk-files/assets/39418668/2aa91971-a4d0-49c4-9290-d54de7a2fa78)
![Screenshot_2](https://github.com/HuskyDG/magisk-files/assets/39418668/5d9308a5-4979-4c49-b844-cf74edb4dfb5)
Screenshot after reboot

**Comment by mgskuser** [comment]
> Don't provide debug logs. Close!

Same issue.
Nox 7.0.5.8
Android version: 12
Magisk version name: d2a66567-delta(26105)-debug
logcat - [magisk-logcat.txt](https://github.com/HuskyDG/magisk-files/files/12420335/magisk-logcat.txt)
magisk log - [magisk_log.log](https://github.com/HuskyDG/magisk-files/files/12420343/magisk_log.log)

**Comment by VisionR1** [comment]
> NoxPlayer does not load custom native bridge so it cannot be fixed
> 
> > 08-23 17:17:13.750  1706  1708 D Magisk  : resetprop: create prop [ro.dalvik.vm.native.bridge]: [libzygisk-ld.so] by property_service
> 
> > 08-23 17:17:16.118  1722  1722 I zygote  : option[46]=-XX:NativeBridge=libnb.so

Can use something like this Riru mod for fix? or modify some file in nox to work? thanks for the try to fix it, i try modify some file inside in the nox player android 12 including in the init/hw folder the init.zygote64_32.rc without success the emulator stuck at boot

![Riru](https://github.com/HuskyDG/magisk-files/assets/25982450/8e0e305b-8024-421d-afda-eaa802098952)

**Comment by HenryKim2022** [comment]
Im install it on android 9 on Nox Player, but the zygisk still not enabled. installing shaminko modules not enabling thats too...

**Comment by mmoadvert** [comment]
Version 25 cannot be installed on the emulator. And in the last one Zygisk does not work. Please fix it. Nox Player on android 12 is the most popular now, you can't go anywhere without it.

**Comment by VisionR1** [comment]
@HuskyDG NoxPlayer A12 finally have zygisk

https://github.com/HuskyDG/magisk-files/issues/199
